doc/mgr/ssh.rst

```diff
@@ -15,6 +15,10 @@ Requirements
 
 - The Python `remoto` library version 0.35 or newer
 
+::
+   # pip3 install remoto
```

Comment:
This file is meant for users of this orchestrator and not meant for develpers. As such I wouldn't recommend users install `remoto` from a different package source. Actually the system package should already handle all dependencies.

Reply:
Including the proper version of remoto (> 0.35)?

Reply:
https://github.com/ceph/ceph/blob/master/ceph.spec.in#L600 but is still blocked by the Python 3 problem we have right now.

Reply:
Ok, I'll drop the `pip` part and leave the version recommendation.